nautobot/dcim/forms.py

```diff
@@ -2686,6 +2728,23 @@ class InterfaceBulkEditForm(
         widget=forms.HiddenInput(),
     )
     enabled = forms.NullBooleanField(required=False, widget=BulkEditNullBooleanSelect)
+    parent = DynamicModelChoiceField(
```

Comment:
Should this be `parent_interface`? If so there are several other spots this needs to be updated it seems.